channels/rdpdr/client/rdpdr_main.c

```diff
@@ -1104,10 +1104,9 @@ static UINT rdpdr_send_client_name_request(rdpdrPlugin* rdpdr)
 	Stream_Write_UINT16(s, PAKID_CORE_CLIENT_NAME); /* PacketId (2 bytes) */
 	Stream_Write_UINT32(s, 1);                      /* unicodeFlag, 0 for ASCII and 1 for Unicode */
 	Stream_Write_UINT32(s, 0);                      /* codePage, must be set to zero */
-	Stream_Write_UINT32(s, (UINT32)computerNameLenW +
-	                           2U); /* computerNameLen, including null terminator */
+	Stream_Write_UINT32(s,
+	                    (UINT32)computerNameLenW); /* computerNameLen, including null terminator */
 	Stream_Write(s, computerNameW, (size_t)computerNameLenW);
-	Stream_Write_UINT16(s, 0); /* null terminator */
```

Comment:
Sure we do not need to write the `\0` terminator?

Reply:
`computerNameW` is 0-terminated so the terminator is already written in the line above.